packages/itwinui-css/src/information-panel/information-panel.scss

```diff
@@ -63,12 +63,11 @@
       }
 
       svg {
-        display: flex;
-        width: $iui-icons-large;
-        height: $iui-icons-large;
+        @include iui-icons-large;
+        @include iui-icons-color;
         margin-right: $iui-s;
+        display: flex;
         flex-shrink: 0;
```

Comment:
should be able to reuse `iui-icons-display` here and get rod of `display` and `flex-shrink` rules

Reply:
`iui-icons-display` uses `display: inline-flex`, not `flex`. This change breaks the test images.

Reply:
I would like to see those changes. Lets meet up about this one.